ard_controller/ard_controller.ino

```diff
@@ -30,6 +37,8 @@ struct JointAngleStruct {
   int left_elbow;
   int right_shoulder;
   int right_elbow;
+  int left_ankle;
```

Comment:
Even though ankle joints are added here, their corresponding states in the state machine are not supposed to be implemented in this PR right?

Reply:
Correct. I added them to the struct but they don't get used anywhere yet.